openfisca_core/tracers.py

```diff
@@ -224,3 +247,25 @@ def print_computation_log(self, aggregate = False):
         """
         for node, depth in self._computation_log:
             self._print_node(node, depth, aggregate)
+
+
+class TracingParameterNodeAtInstant():
```

Comment:
I'd give it a more classy name, depending of what we're doing here. Either

- An actual data structure => `ParameterNodeAtInstantTrace` (noun)
- A command to do something => `TraceParameterNodeAtInstant` (verb, imperative)

Reply:
"Tracing" here is an adjective, it suggests a modification to the behaviour of the wrapped class. The class is not a data structure (it doesn't hold the trace, the Tracer does), nor is it a command (it doesn't have an `execute()` method or the like). This name is fine, stet.

Reply:
Ahhh you're using `tracing` as a noun!